workflows/gen_crates.py

```diff
@@ -56,6 +62,113 @@
 PLANEMO_VERSION = f">={planemo.__version__}"
 PLANEMO_TEST_SUFFIXES = ["-tests", "_tests", "-test", "_test"]
 PLANEMO_TEST_EXTENSIONS = [".yml", ".yaml", ".json"]
+HUB_URL = "https://workflowhub.eu"
+HUB_API_KEY = os.getenv("HUB_API_KEY")
```

Comment:
Can we do this without the API key and pushing from our side? I'd prefer a github app like we've installed for dockstore, that would make the repository more portable across organizations or owners.